src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -70,35 +71,14 @@ public AlgorithmId(uint flags)
         // From ECMAKey.h
         private static readonly ImmutableArray<byte> s_ecmaKey = ImmutableArray.Create(new byte[] { 0, 0, 0, 0, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0 });
 
-        // From strongname.h
-        // Note: different from the PUBLICKEYBLOB specified in wincrypt.h. The 
```

Comment:
Please keep this comment somewhere so there's documentation on where these structures come from.